applications/vr-grabscale/VRBuildGrabScale.js

```diff
@@ -0,0 +1,496 @@
+//
+//  VRBuildGrabScale.js
+//  
+//  Created by Basinsky on 2/2/2020
```

Comment:
Ambiguous date format - 2 Feb or Feb 2 - though in this case both interpretations are the same.